build.rs

```diff
@@ -49,7 +49,12 @@ fn main() -> Result<()> {
     let target = env::var("TARGET").expect("TARGET was not set");
     let mut base_config = cc::Build::new();
 
-    language_std(&mut base_config, "c++17");
+    // try to match the standards expected by the compilers in GitHub Actions images, as of October 2021
+    if target.contains("windows") {
+        language_std(&mut base_config, "c++latest");
+    } else {
+        language_std(&mut base_config, "c++17");
+    }
```

Comment:
Ok, that isn't going to work.
```suggestion
    language_std(&mut base_config, "c++17");
```